### PR TITLE
Case 21943: Fix transparent shape flickering (RC81)

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/21943/Alpha-Transparencies-Z-Fighting

Test plan: https://github.com/highfidelity/hifi/pull/15273

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18426/HighFidelity-Beta-PR15272-2267942.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18426/HighFidelity-Beta-Interface-PR15272-2267942.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18426/HighFidelity-Beta-PR15272-2267942.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18426/HighFidelity-Quest-Beta-PR15272-2267942.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18426/HighFidelity-Beta-PR15272-2267942.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18426/HighFidelity-Beta-Interface-PR15272-2267942.exe

File: libraries/entities-renderer/src/RenderableShapeEntityItem.cpp [hunk 1]
```diff
@@ -281,9 +281,9 @@ void ShapeEntityRenderer::doRender(RenderArgs* args) {
         outColor.a *= _isFading ? Interpolate::calculateFadeRatio(_fadeStartTime) : 1.0f;
         render::ShapePipelinePointer pipeline;
         if (renderLayer == RenderLayer::WORLD) {
-            pipeline = GeometryCache::getShapePipeline(false, outColor.a < 1.0f, true, false);
+            pipeline = outColor.a < 1.0f ? geometryCache->getTransparentShapePipeline() : geometryCache->getOpaqueShapePipeline();
```
Comment (SamGondelman) [inline libraries/entities-renderer/src/RenderableShapeEntityItem.cpp]: Nothing new here, just reverting some inconsequential changes from these PRs:
https://github.com/highfidelity/hifi/pull/15197
https://github.com/highfidelity/hifi/pull/15257

**Pull request merged**